Origin: EmberBSD contributors, AI-assisted. Not submitted upstream.
Rebuild the installed archive index with the portable ranlib interface.
LLVM ranlib does not accept the BSD/GNU symbol-map timestamp -t option.
Apply the same operation to the generic and platform-specific templates.

--- a/pkgtools/bootstrap-mk-files/Makefile
+++ b/pkgtools/bootstrap-mk-files/Makefile
@@ -1,7 +1,7 @@
 # $NetBSD: Makefile,v 1.62 2025/06/01 19:07:23 triaxx Exp $
 
 PKGNAME=	bootstrap-mk-files-20250601
-PKGREVISION=	4
+PKGREVISION=	6
 CATEGORIES=	pkgtools
 
 CONFLICTS+=	mk-files-[0-9]*
--- a/pkgtools/bootstrap-mk-files/files/bsd.lib.mk
+++ b/pkgtools/bootstrap-mk-files/files/bsd.lib.mk
@@ -356,7 +356,7 @@
 __archiveinstall: .USE
 	${INSTALL} ${RENAME} ${PRESERVE} ${COPY} ${INSTPRIV} -o ${LIBOWN} \
 	    -g ${LIBGRP} -m 600 ${.ALLSRC} ${.TARGET}
-	${RANLIB} -t ${.TARGET}
+	${RANLIB} ${.TARGET}
 	chmod ${LIBMODE} ${.TARGET}
 
 DPSRCS+=	${SRCS:M*.l:.l=.c} ${SRCS:M*.y:.y=.c}
--- a/pkgtools/bootstrap-mk-files/files/mods/Cygwin.bsd.lib.mk
+++ b/pkgtools/bootstrap-mk-files/files/mods/Cygwin.bsd.lib.mk
@@ -308,7 +308,7 @@
 __archiveinstall: .USE
 	${INSTALL} ${RENAME} ${PRESERVE} ${COPY} ${INSTPRIV} -o ${LIBOWN} \
 	    -g ${LIBGRP} -m 600 ${.ALLSRC} ${.TARGET}
-	${RANLIB} -t ${.TARGET}
+	${RANLIB} ${.TARGET}
 	chmod ${LIBMODE} ${.TARGET}
 
 DPSRCS+=	${SRCS:M*.l:.l=.c} ${SRCS:M*.y:.y=.c}
--- a/pkgtools/bootstrap-mk-files/files/mods/Haiku.bsd.lib.mk
+++ b/pkgtools/bootstrap-mk-files/files/mods/Haiku.bsd.lib.mk
@@ -352,7 +352,7 @@
 __archiveinstall: .USE
 	${INSTALL} ${RENAME} ${PRESERVE} ${COPY} ${INSTPRIV} -o ${LIBOWN} \
 	    -g ${LIBGRP} -m 600 ${.ALLSRC} ${.TARGET}
-	${RANLIB} -t ${.TARGET}
+	${RANLIB} ${.TARGET}
 	chmod ${LIBMODE} ${.TARGET}
 
 DPSRCS+=	${SRCS:M*.l:.l=.c} ${SRCS:M*.y:.y=.c}
--- a/pkgtools/bootstrap-mk-files/files/mods/Linux.bsd.lib.mk
+++ b/pkgtools/bootstrap-mk-files/files/mods/Linux.bsd.lib.mk
@@ -356,7 +356,7 @@
 __archiveinstall: .USE
 	${INSTALL} ${RENAME} ${PRESERVE} ${COPY} ${INSTPRIV} -o ${LIBOWN} \
 	    -g ${LIBGRP} -m 600 ${.ALLSRC} ${.TARGET}
-	${RANLIB} -t ${.TARGET}
+	${RANLIB} ${.TARGET}
 	chmod ${LIBMODE} ${.TARGET}
 
 DPSRCS+=	${SRCS:M*.l:.l=.c} ${SRCS:M*.y:.y=.c}
--- a/pkgtools/bootstrap-mk-files/files/mods/Minix.bsd.lib.mk
+++ b/pkgtools/bootstrap-mk-files/files/mods/Minix.bsd.lib.mk
@@ -352,7 +352,7 @@
 __archiveinstall: .USE
 	${INSTALL} ${RENAME} ${PRESERVE} ${COPY} ${INSTPRIV} -o ${LIBOWN} \
 	    -g ${LIBGRP} -m 600 ${.ALLSRC} ${.TARGET}
-	${RANLIB} -t ${.TARGET}
+	${RANLIB} ${.TARGET}
 	chmod ${LIBMODE} ${.TARGET}
 
 DPSRCS+=	${SRCS:M*.l:.l=.c} ${SRCS:M*.y:.y=.c}
--- a/pkgtools/bootstrap-mk-files/files/mods/OpenBSD.bsd.lib.mk
+++ b/pkgtools/bootstrap-mk-files/files/mods/OpenBSD.bsd.lib.mk
@@ -356,7 +356,7 @@
 __archiveinstall: .USE
 	${INSTALL} ${RENAME} ${PRESERVE} ${COPY} ${INSTPRIV} -o ${LIBOWN} \
 	    -g ${LIBGRP} -m 600 ${.ALLSRC} ${.TARGET}
-	${RANLIB} -t ${.TARGET}
+	${RANLIB} ${.TARGET}
 	chmod ${LIBMODE} ${.TARGET}
 
 DPSRCS+=	${SRCS:M*.l:.l=.c} ${SRCS:M*.y:.y=.c}
--- a/pkgtools/bootstrap-mk-files/files/mods/SCO_SV.bsd.lib.mk
+++ b/pkgtools/bootstrap-mk-files/files/mods/SCO_SV.bsd.lib.mk
@@ -356,7 +356,7 @@
 __archiveinstall: .USE
 	${INSTALL} ${RENAME} ${PRESERVE} ${COPY} ${INSTPRIV} -o ${LIBOWN} \
 	    -g ${LIBGRP} -m 600 ${.ALLSRC} ${.TARGET}
-	${RANLIB} -t ${.TARGET}
+	${RANLIB} ${.TARGET}
 	chmod ${LIBMODE} ${.TARGET}
 
 DPSRCS+=	${SRCS:M*.l:.l=.c} ${SRCS:M*.y:.y=.c}
